mifosng-provider/src/main/java/org/mifosplatform/portfolio/charge/domain/ChargeCalculationType.java

```diff
@@ -9,9 +9,10 @@
 
     INVALID(0, "chargeCalculationType.invalid"), //
     FLAT(1, "chargeCalculationType.flat"), //
-    PERCENT_OF_AMOUNT(2, "chargeCalculationType.percent.of.amount"), //
+    PERCENT_OF_APPROVED_AMOUNT(2, "chargeCalculationType.percent.of.amount"), //
     PERCENT_OF_AMOUNT_AND_INTEREST(3, "chargeCalculationType.percent.of.amount.and.interest"), //
```

Comment:
what amount refers to ? is it approved or disbursed? do we need to  support both(approval and disbursement) the types?